neonvm/runner/main.go

```diff
@@ -588,7 +602,7 @@ func main() {
 			}
 			qemuCmd = append(qemuCmd, "-drive", fmt.Sprintf("id=%s,file=%s,if=virtio,media=disk,cache=none%s", disk.Name, dPath, discard))
 		case disk.ConfigMap != nil || disk.Secret != nil:
-			dPath := fmt.Sprintf("%s/%s.qcow2", mountedDiskPath, disk.Name)
+			dPath := fmt.Sprintf("%s/%s.iso", mountedDiskPath, disk.Name)
```

Comment:
why this change? if unrelated, let's make it a separate PR (this one is already quite complex in terms of number of files / points in code changed)

Reply:
Opened #735